### ISSUE TITLE
unpickler fails with UnicodeDecodeError

### ISSUE THREAD

**Issue by yshui** [apitrace/apitrace#838]
See https://github.com/yshui/picom/issues/892#issuecomment-1321259167, this trace file contains gl string marker which is utf8 encoded. while loading the trace, unpickler fails with UnicodeDecodeError

adding `encoding = 'utf-8'` to `pickle.load` fixes the problem for this particular trace file, but I am not sure if it's the correct solution.

**Comment by jrfonseca** [comment]
I downloaded the picom.trace referred on that issue, but didn't come across any `UnicodeDecodeError`:

```
$ python3 scripts/leaks.py issues/838/picom.trace 
43: error: texture 1 was not destroyed until 17481926
50: error: texture 2 was not destroyed until 17481926
1148921: error: texture 33 was not destroyed until 17481926
6529929: error: texture 24 was not destroyed until 17481926
7656234: error: texture 44 was not destroyed until 17481926
7695128: error: texture 42 was not destroyed until 17481926
8140876: error: texture 34 was not destroyed until 17481926
14221207: error: texture 35 was not destroyed until 17481926
17355458: error: texture 31 was not destroyed until 17481926
17380351: error: texture 41 was not destroyed until 17481926
17465807: error: texture 39 was not destroyed until 17481926
```

Can you confirm you're using a recent apitrace checkout?

**Comment by yshui** [comment]
@jrfonseca i am using what's available from my distro, which i realize it's quite old.

let me try again.

**Comment by yshui** [comment]
can confirm it's fixed in latest version. 

i should've checked the version before reporting :sweat_smile:

**Comment by jrfonseca** [comment]
Thanks for confirming.
